stream_reader.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	headerData  = []byte("data: ")
+	headerData  = []byte("data:")
```

Comment:
can we pass a provider name to this library, and do a condition if the provider is lepton we dont do this headerData, and if its tensoropera we will do this format.

Reply:
yeah I can check

Reply:
There is another option. Apply this change and fix the code to trim possible space after removing headerData prefix.